Documentation/ryujit-overview.md

```diff
@@ -18,11 +18,11 @@ The first objective was the primary motivation for evolving the existing code ba
 
 RyuJIT provides the just in time compilation service for the .NET runtime. The runtime itself is variously called the EE (execution engine), the VM (virtual machine) or simply the CLR (common language runtime). Depending upon the configuration, the EE and JIT may reside in the same or different executable files. RyuJIT implements the JIT side of the JIT/EE interfaces:
 
-* ICorJitCompiler – this is the interface that the JIT compiler implements. This interface is defined in src\inc\corjit.h and its implementation is in [src/jit/ee_il_dll.cpp](https://github.com/dotnet/coreclr/blob/master/src/jit/ee_il_dll.cpp). The following are the key methods on this interface:
-  * compileMethod is the main entry point for the JIT. The EE passes it a ICorJitInfo object, and the “info” containing the IL, the method header, and various other useful tidbits. It returns a pointer to the code, its size, and additional GC, EH and (optionally) debug info.
-  * getVersionIdentifier is the mechanism by which the JIT/EE interface is versioned. There is a single GUID (manually generated) which the JIT and EE must agree on.
-  * getMaxIntrinsicSIMDVectorLength communicates to the EE the largest SIMD vector length that the JIT can support.
-* ICorJitInfo – this is the interface that the EE implements. It has many methods defined on it that allow the JIT to look up metadata tokens, traverse type signatures, compute field and vtable offsets, find method entry points, construct string literals, etc. This bulk of this interface is inherited from ICorJitDynamicInfo which is defined in src\inc\corinfo.h. The implementation is defined in src\vm\jitInterface.cpp.
+* `ICorJitCompiler` – this is the interface that the JIT compiler implements. This interface is defined in [src\inc\corjit.h](https://github.com/dotnet/coreclr/blob/master/src/inc/corjit.h) and its implementation is in [src/jit/ee_il_dll.cpp](https://github.com/dotnet/coreclr/blob/master/src/jit/ee_il_dll.cpp). The following are the key methods on this interface:
```

Comment:
There are a few other places in the doc where the path separator is mixed like here, i.e. `src\inc\corjit.h` vs. `src/jit/ee_il_dll.cpp`. Would be good to decide on one format and apply it everywhere.